### DISCUSSION TITLE
Add block and show element in block grid list - don't open it

### DISCUSSION THREAD

**Discussion by jesperordrup** [umbraco/Umbraco-CMS#14836] [Features and ideas]
When you add blocks to a block grid list (bgl) it opens and allows you to fill in properties. But if your block defines areas you might want not to open the block for editing but instead show the bgl with the added block and the areas to continue adding stuff to areas.

This should be a config option for the block how it should act.

**Comment by jesperordrup** [comment]
Forgot to mention that there is a feature to hide the content editor. When doing so the editor the behavior changes to just inserting the block. 

The only problem is that later you cant open the editor by clicking the inserted block in the blocklist. 

<img width="555" alt="image" src="https://github.com/umbraco/Umbraco-CMS/assets/3120151/2d7d2335-2a89-4937-95a7-27cf4cb336e1">

Suggestion:

Add a new option: "Allow open editor from block"

**Comment by nielslyngsoe** [comment]
Hi @jesperordrup 

Thanks for bringing your perspective and idea.

I'm a fan of consistency, as that helps the users understand how things work, whereas less consistency makes it harder to see the pattern. In other words, an extra click for user confirmation can be better in some cases than a fast workflow. So before we make things inconsistent I would like to understand the case a bit more in depth.

As in, the use case where the user feels pain from this confirmation of the specific block.
Ideally screenshots and a description of the user flow and user goal. So I can understand why the user would feel 'interrupted' or 'distracted' by the Block opening for editing.

Thanks 🙂

Btw. a little tip, which I'm not sure helps in your case, is that you can hold CTRL? + click if you like to add multiple Blocks. Well, I didn't test it now, so that might be another key than CTRL.

**Comment by jesperordrup** [comment]
Your welcome @nielslyngsoe 

About consistency. I agree that it's an important goal. I have some observations:

You can already disable showing the editor in the flow, completely. Then after inserting, you can't open the editor. I'm actually not sure if this is intentional, but I can make sense out of it in situations where the block element has no properties to show.

Generally, I encourage the idea of making the big picture and then dive into details. This feels right when setting up a layout, creating a document, creating content etc. Taking this to areas. I much prefer to insert some blocks, arrange them and then have a good look. And then fill in the blanks.

Back to consistency: Since you already can disable the editor its already possible to break consistency.  Either way it feels right to say that Its up to the implementer to make this consistent throughout the specific solution. Maybe some block types consistently are "area first" and others not.

My suggestion to add another option for allowing to open editor after adding will make current and suggested scenario possible without breaking current functionality.
